books/free-programming-books-pt_BR.md

```diff
@@ -256,6 +256,7 @@
 * [Guia JavaScript](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Guide)
 * [Guia Rápido de Desenvolvimento para Firefox OS](https://leanpub.com/guiarapidofirefoxos/read) (HTML)
 * [JS: A forma certa](http://jstherightway.org/pt-br/)
+* [Teach Yourself Javascript](https://vulms.vu.edu.pk/Courses/CS202/Downloads/JavaScript%20Book.pdf) (PDF)
```

Comment:
This is not an authorized distribution. Refer to docs/CONTRIBUTING.md for guidelines.